examples/basic/lines/line_missing_points.py

```diff
@@ -6,5 +6,8 @@
 
 # add a line renderer with NaN values
 p.line([1, 2, 3, nan, 4, 5], [6, 7, 2, 4, 4, 5], line_width=2)
+# don't use native None in a line renderer, because it will be drawn as 0
+p.line([1, 2, 3, None, 4, 5], [6, 7, 2, 4, 4, 5], line_width=2,
+       line_dash="dashed", color="red")
```

Comment:
suggest vertical whitespace to make the comments easier to read
```suggestion

# don't use native None in a line renderer, because it will be drawn as 0
p.line([1, 2, 3, None, 4, 5], [6, 7, 2, 4, 4, 5], line_width=2,
       line_dash="dashed", color="red")
```